src/inputs/Select/index.tsx

```diff
@@ -64,7 +81,16 @@ function Select({ items, activeItemId, onItemClick, id, ...rest }: Props) {
             return (
               <MenuItem value={i.id} key={i.id}>
                 {i.iconUrl && <IconImg alt={i.label} src={i.iconUrl} />}
-                <span>{i.label}</span>
+                <div>
+                  {i.title && (
```

Comment:
why is type of `title` is `string` in `type SelectItem` but it's only used as a boolean? should it be boolean instead or you forgot to display it somewhere?

Reply:
good catch